layers/provision/zot.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=OCI Distribution Registry
+After=network.target local-fs.target
```

Comment:
I think you want `After=network-online.target` -- network.target happens right when networking services (systemd-networkd, etc) start -- `network-online.target` is reached after any configured interfaces are up (or we timeout waiting for them to come up).

Reply:
Alternatively, if zot listens and binds to 0.0.0.0 then it can see when any interface/ip comes up.  This is what ssh does.  Though if zot needs changes to work then waiting for `network-online.target` is best.

Reply:
Actually I've removed this file.  Firstly it was in the wrong place (provision/ rather than install/) and secondly we want to lxc-execute zot, not lxc-start a full system(d) container with zot.service.

But thanks, indeed that's what i would have wanted